### DISCUSSION TITLE
Automatic flight log book

### DISCUSSION THREAD

**Discussion by MWunderer** [EdgeTX/edgetx#2128] [Ideas]
As actually discuss on german frsky-forum, an automatic flight log book was a really nice fetaure. For example the Vbar radios it allready has it. 
What should it do: a global .csv (for example flightbook.csv) in which the radio saves every flight. On model settings (may on screen one with the timers) can be defined a trigger variable (logic stwich, switch etc.) 'flight start/stop' or so. On rising edge(high level the flight starts, on falling edge/low level the flight ends. In flight book will be saved automatically the date, start-time, end-time, duration and model name. Thats it. So afterwards in one single file you have the information, how many flights are made, which models will be used etc.
Similar solutions was allready made in some way via lua, but not as smooth and simjple to use. I think, a feture like this can be interessting for everyone.

**Comment by scooper1** [comment]
It would be nice to extend to include minimum values like battery voltage, rssi

**Comment by mason-home** [comment]
很好，特别需要这个功能，一直没找到好的解决方法，尤其我现在使用的黑白屏

**Comment by inventor7777** [comment]
This sort of exists as "SD Logs" but it's super basic. This sounds like a pretty good idea, especially if it can include min/max values
